.browserslistrc

```diff
@@ -0,0 +1 @@
+supports es6-module and last 3 years
```

Comment:
@AnthorNet set this to what browsers you want to support.

I set it to a pretty conservative "all browser versions released in the last 3 years". This can probably be reduced to just the last year or even 6 months as I don't think there would be many users in the app's target audience that are using older browsers.

If you have analytics set up for the site, using that data would be a good way work out the best browsers to support.

Reply:
I've done some testing and it seems like `since 2021-07-01` is the best we can do without sacrificing performance.
Going back further than that seems to caused my save game load time to jump from ~50s to ~70s

Reply:
Why adding that if it changes stuff in the code?

Reply:
You mean adding babel?

Reply:
Yep, not sure what it does so...? :D

Reply:
Babel is design to down-level newer js code to older js code that performs the same task. By using babel it allows us to program using the latest js features but still have the code work on slightly older browser that don't yet support those js features.

If everyone is using an up to date browser, then there would be no need for babel but not everyone does hence it's good to have a bit of backwards compatibility.

Reply:
If you only want to target the latest browser, you could set this to something like `last 2 versions`; which going by Chrome's release schedule would equate to "support browser versions released in the last 8 weeks"

Reply:
It seems that down-levelling the private class fields was what was causing the performance degradations. This would of been because babel would have being ensuring that the fields were not readable from outside the class and thus would have been doing a bunch of extra work to make that the case.

With #192 closed, there isn't any performance hits and so we can keep support at "last 3 years" if you like.